plugin.video.otaku/resources/lib/AniListBrowser.py

```diff
@@ -2754,7 +2754,13 @@ def get_upcoming_next_season_upcoming(self, page=1, format_in=''):
         return self._process_anilist_view(upcoming, "anilist_upcoming_next_season_upcoming/%d", page)
 
     def get_airing(self, page=1, format_in=''):
-        airing = database.get(self._get_airing, 12, page, self.format_in_type)
+        dbargs = {}
+        calendarRefresh = control.getGlobalProp("calendarRefresh")
+        if calendarRefresh == True:
+            dbargs['otaku_reload'] = True
+            control.setGlobalProp("calendarRefresh", False)
+
```

Comment:
Could reduce this all down to:
dbargs = { "otaku_reload" : control.getGlobalProp("calendarRefresh") }
control.setGlobalProp("calendarRefresh", False)